docs/trace/building-your-own-processor/MyActivityProcessor.cs

```diff
@@ -22,29 +22,37 @@
 
 internal class MyActivityProcessor : ActivityProcessor
 {
-    public MyActivityProcessor()
+    private readonly string name;
+
+    public MyActivityProcessor(string name)

```

Comment:
Need to take the next ActivityProcessor, and call its corresponding methods in Start/End/etc.

Reply:
For demonstration I was trying to make it simple by not chaining the processors at all.